### DISCUSSION TITLE
Change H3 Hexagon Elevation Variable ?

### DISCUSSION THREAD

**Discussion by divyselva** [visgl/deck.gl#5335] [Q&A]
Is there a way to connect the Hexagon layers elevation value to represent a different column/variable other than the count of points? If so how would one go about doing this, or are there any examples to look at.

**Comment by Pessimistress** [comment]
[getElevationValue](https://deck.gl/docs/api-reference/aggregation-layers/hexagon-layer#getelevationvalue)

**Reply by divyselva** [reply]
I've tried this, but it returns the same result as not specifying it. How would i get it to return the average or mean of the points instead of the length? Thanks

**Reply by Pessimistress** [reply]
What exactly did you try?

**Reply by divyselva** [reply]
![Screen Shot 2021-01-14 at 6 40 50 PM](https://user-images.githubusercontent.com/58446818/104662092-0858ff00-5698-11eb-9966-b8ffea277fad.png)

This is what i have tried. I've also tried it without the elevationAggregation line

**Reply by Pessimistress** [reply]
Please read the examples on that page, under `elevationAggregation`. `getElevationValue` does not receive an individual data object as argument.

**Reply by divyselva** [reply]
So getElevationValue won't receive that column individually? Is it because the data format is from a csv?

**Reply by Pessimistress** [reply]
>  It takes an array of points inside each bin as arguments, returns a number. 

You can log the argument to see its content.
